pkg/client/dubbo/dubbo.go

```diff
@@ -22,20 +22,20 @@ import (
 	"strings"
 	"sync"
 	"time"
-)
 
-import (
+	"go.opentelemetry.io/otel/trace"
+
 	"github.com/apache/dubbo-go/common/constant"
 	dg "github.com/apache/dubbo-go/config"
 	"github.com/apache/dubbo-go/protocol/dubbo"
-	fc "github.com/dubbogo/dubbo-go-pixiu-filter/pkg/api/config"
-	"github.com/pkg/errors"
-)
+	"go.opentelemetry.io/otel"
```

Comment:
pls split this import block into three part:
- first block contains all Go build-in package;
- second block contains third party package;
- third block contains package in project

Reply:
okay

Reply:
One extra question, why not use goimports or goreturn of the go community?

Reply:
we can use tool like this https://github.com/apache/dubbo-go/pull/1346 later